packages/react/src/components/uiComponents.tsx

```diff
@@ -36,7 +36,7 @@ class Portal extends React.PureComponent<MountProps, {}> {
   private portalRef = React.createRef<HTMLDivElement>();
 
   componentDidUpdate(prevProps: Readonly<MountProps>) {
-    if (prevProps.props.appearance !== this.props.props.appearance) {
+    if (prevProps.props.appearance !== this.props.props.appearance || prevProps.props !== this.props.props) {
```

Comment:
❓ Can this be simplified to this?
```suggestion
    if (prevProps.props !== this.props.props) {
```

Reply:
Hey John, sorry if I wasn't clear enough. This snippet is not showing a nested if, its showing a diff between the red/green parts.
So to rephrase my question, is the `if (prevProps.props !== this.props.props)`  enough?